RecoTracker/TkNavigation/test/NavigationSchoolAnalyzer_cfg.py

```diff
@@ -4,7 +4,7 @@
 
 process = cms.Process("NavigationSchoolAnalyze")
 
-# process.load("Configuration.StandardSequences.Geometry_cff")
+#process.load("Configuration.StandardSequences.GeometryDB_cff")
 #process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
```

Comment:
No, because as an example this file is now configured for phase-2 (which takes the geometry from XML), but in general it needs to come from the DB....